ChangeLog.md

```diff
@@ -4,6 +4,12 @@ SPDX-License-Identifier: MIT
 -->
 # Changelog for stablecoin
 
+## Unreleased
+
+* [#85](https://github.com/tqtezos/stablecoin/pull/92)
+  Removed Permissions_descriptor entrypoint.
+  Changed owner hook behavior to `Owner_no_hook`.
```

Comment:
I suppose it lacks the removal of `total_supply`.

Reply:
I didn't include it because it was an internal change. Will include it now.

Reply:
Well, it's not internal at least because it's also removed from the client.

Reply:
Ah. Right. Have included now.